### DISCUSSION TITLE
When was alt.selection_point added?

### DISCUSSION THREAD

**Discussion by freddyaboulton** [vega/altair#3085] [Q&A] [answered]
Hi!

I am a maintainer of a library that uses altair. 

I recently noticed the following warnings:

```
AltairDeprecationWarning: 'selection' is deprecated.
   Use 'selection_point()' or 'selection_interval()' instead; these functions also include more helpful docstrings.
```

```
AltairDeprecationWarning: 'add_selection' is deprecated. Use 'add_params' instead.
  warnings.warn(message, AltairDeprecationWarning, stacklevel=1)
```

When did `add_params` and `selection_points` get added to the library? Do they only work in altair 5.0+?

**Comment by mattijn** [comment, marked as answer]
Thanks for asking @freddyaboulton! You are right. These changes were introduced in version Altair 5 and do not work in prior versions. 

You can read more about the changes in the release notes within the docs: https://altair-viz.github.io/releases/changes.html#version-5-0-0-released-may-9-2023. 

If you are also interested in the associated Pull Requests you also can check the release notes within GitHub release page: https://github.com/altair-viz/altair/releases/tag/v5.0.0.

**Reply by freddyaboulton** [reply]
Thank you @mattijn !
